packages/fuel-indexer-tests/components/indices/fuel-indexer-test/src/lib.rs

```diff
@@ -139,7 +139,7 @@ mod fuel_indexer_test {
             id: result,
             result,
             gas_used,
-            blob: vec![1u8, 1, 1, 1, 1],
+            blob: Blob(vec![1u8, 1, 1, 1, 1]),
```

Comment:
Should not have to wrap a `Blob` here. Using proper `impl` we can just `.into()` this